src/main/java/graphql/testrunner/repository/TestResultRepo.java

```diff
@@ -0,0 +1,43 @@
+package graphql.testrunner.repository;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TestResultRepo {
```

Comment:
can you think of a better name for this class, `FirestoreWriter` `TestResultSaver` ...?